pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
Sorry, that was not what I meant. I would just leave it as it was (we had a discussion about this before, and explicitly change arrays no longer to evaluate equal to Index objects). The behaviour was already good, my only question was about to keep the test (but with updated syntax of course)